src/entities/dataset/file-import.ts

```diff
@@ -47,9 +47,6 @@ export class FileImport extends BaseEntity {
     @Column({ type: 'enum', enum: Object.values(ImportType), nullable: false })
     type: ImportType;
 
-    @Column({ type: 'enum', enum: Object.values(DataLocation), nullable: false })
-    location: DataLocation;
```

Comment:
Are we able to remove this enum now as well?

Reply:
Yes we can remove the enum as well... Let clean that up.

Reply:
This is gone now as part of the last push.